app.js

```diff
@@ -1,15 +1,21 @@
 const express = require("express");
-const { getTopics } = require("./controllers/newsControllers");
+const { getTopics, getArticleById } = require("./controllers/newsControllers");
+const {
+  invalidEndpoint,
+  serverError,
+  PSQLerrors,
+  costumError,
+} = require(`./controllers/errorHandling.controller`);
```

Comment:
Awesome work separating out into their own error controller, one small thing, it's spelled "custom" errors